design-library/src/components/BccLinkItem/BccLinkItem.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, it, expect } from "vitest";
+
+import { mount } from "@vue/test-utils";
+import BccLinkItem from "./BccLinkItem.vue";
+
+describe("BccLinkItem", () => {
+  it("renders", async () => {
+    const wrapper = mount(BccLinkItem, {
+      props: {},
+      slots: {
+        default: "Title..",
+      },
+    });
+    expect(wrapper.html()).toContain("Title..");
+    expect(wrapper.html()).toContain("h-4 w-4");
+    expect(wrapper.html()).not.toContain("border-none");
+  });
+
+  it("renders last element", async () => {
+    const wrapper = mount(BccLinkItem, { props: { isLast: true } });
+    expect(wrapper.html()).toContain("h-4 w-4");
+    expect(wrapper.html()).toContain("border-none");
+  });
```

Comment:
I don't think testing for specific classes is a good idea because it makes our tests very tightly coupled to the implemented markup - maybe only if they are named classes from our CSS but then still... I think for these "visual" cases just a Storybook story is enough and no test is needed.